yt/yt/core/http/retriable_client.cpp

```diff
@@ -0,0 +1,210 @@
+#include "config.h"
+
+#include "retriable_client.h"
+#include "private.h"
+
+#include <yt/yt/core/http/client.h>
+#include <yt/yt/core/http/helpers.h>
+#include <yt/yt/core/http/public.h>
+#include <yt/yt/core/json/json_parser.h>
+
+namespace NYT::NHttp {
+
+using namespace NNet;
+using namespace NYTree;
+using namespace NConcurrency;
+
+static const auto& Logger = HttpLogger;
+
+////////////////////////////////////////////////////////////////////////////////
+
+class TJsonResponseChecker
+    : public IResponseChecker
+{
+public:
+    TJsonResponseChecker(
+        TJsonErrorChecker errorChecker,
+        NJson::TJsonFormatConfigPtr jsonFormatConfig
+    )
+    : JsonFormatConfig_(std::move(jsonFormatConfig))
+    , ErrorChecker_(std::move(errorChecker))
+    { }
+
+    TError CheckError(const IResponsePtr& response) override {
```

Comment:
We write opening brace of function body on the new line, fix all occurencies, please